packages/dmn-js-shared/src/components/InputSelect.js

```diff
@@ -89,10 +89,15 @@ export default class InputSelect extends Component {
     const container = this.renderer.getContainer();
 
     container.appendChild(this._portalEl);
+
+    // suppress mousedown event propagation to handle click events inside the component
+    this._portalEl.addEventListener('mousedown', stopPropagation);

```

Comment:
Why are we not handling this behavior in the context menu component which is the one closing itself on click?

Reply:
The context menu listens to [mousedown events now](https://github.com/bpmn-io/table-js/blob/master/src/features/context-menu/components/ContextMenuComponent.js#L213). The click event fires always after mousedown, so the context menu is already closed when the click can be handled.